README.md

```diff
@@ -28,7 +28,7 @@ If this project was received as part of an evaluation, then the ``.use-pdfjt-lm`
 
 ## Building from the command line
 
-This project includes [maven-wrapper](https://github.com/datalogics/maven-wrapper), which fully encapsulates the build system required for PDF Java Toolkit. To build with Maven, simply type ``./mvnw`` at the command line (``.\mvnw`` on Windows).
+This project includes [maven-wrapper](https://github.com/datalogics/maven-wrapper), which fully encapsulates the build system required for PDF Java Toolkit. To build with Maven, simply type ``./mvnw verify`` at the command line (``.\mvnw verify`` on Windows).
```

Comment:
I would love to add more clarity to this explanation, but I feel like we ought to discuss what the user is going to see once they've done everything in the README. What should the folder look like? How would they fire up a sample to do something? I'm a developer on this thing, and I have to confess its a little confusing right now even to me.

Rather than getting into a cycle of me tweaking the text, pushing the change, and waiting for your comments, let's sit down and decide what we'd like it to say.

I'll also point out that our documentation writer has already contributed changes to this file, and his input may be useful.